### PR TITLE
Add extension - "The Office" Quotes

### PR DESCRIPTION
**Pull request by sandypockets**
## Description

- Added a new extension
- Extension allows users to search, view, and copy/paste top quotes from the TV series "The Office". 
- Users can shuffle quotes
- Users can view the next quote in the list (from the Detail view)

## Screencast

Screenshots included in the README [here](https://github.com/sandypockets/extensions/blob/main/extensions/office-quotes/README.md).

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [] I checked that assets used by the `README` are placed in the `metadata` folder

> I checked that assets used by the `README` are placed in the `metadata` folder

Is this a new structure? I submitted a PR previously and used a `media` folder, which is what I've done here. Is that going to be OK, or would you prefer an updated PR with the `metadata` folder?

Thanks!

### CODE REVIEW

File: extensions/office-quotes/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  }, []);
+
+  return (
+    <List isLoading={loading}>
+      {quotes?.length > 0 &&
+        quotes.map((quote, index) => (
+          <List.Item
+            title={quote["content"]}
+            key={quote["content"]}
+            accessories={[
```
Comment (anhthang) [inline extensions/office-quotes/src/index.tsx:33]:
```
accessories={[
    { text: `${quote["character"]["firstname"]} ${quote["character"]["lastname"]}` },
    { icon: Icon.Person },
]}
```

I think you can do this, so Person icons are aligned

Comment (sandypockets) [reply extensions/office-quotes/src/index.tsx:33]: Good call, thank you! I will make the adjustment today.

**Comment by sandypockets** [comment]
The recent commit fixes the alignment of the Person icons. Example in the screenshot below.

Screen Shot 2022-03-31 at 9 13 46 PM

**Review by anhthang — APPROVED** [review]

**Comment by sandypockets** [comment]
Should be all set on my end

File: extensions/office-quotes/package.json [hunk 2]
```diff
@@ -5,18 +5,25 @@
...
+    "Fun"
+  ],
   "license": "MIT",
   "commands": [
     {
       "name": "index",
-      "title": "Office Quote",
-      "description": "Fetch a random quote from the Office API",
+      "title": "'The Office' Quotes",
+      "subtitle": "Search, view, and copy/paste",
```
Comment (fe9lix) [inline extensions/office-quotes/package.json:19]:
```suggestion
      "subtitle": "",
```

**Comment by fe9lix** [comment]
@sandypockets Cheers, I've removed the subtitle (since it's too generic and doesn't add meaningful keywords to the search) and added action icons (in an ActionPanel either all items should have icons or none of them, according to our store guidelines)

**Pull request merged**